app/controllers/application_controller.rb

```diff
@@ -3,4 +3,12 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
```

Comment:
Don't think we'll need to do that :) See other PR -> https://github.com/LandRegistry/Property-Frontend/pull/18